chainercv/datasets/__init__.py

```diff
@@ -19,6 +19,13 @@
 from chainercv.datasets.cub.cub_utils import cub_label_names  # NOQA
 from chainercv.datasets.directory_parsing_label_dataset import directory_parsing_label_names  # NOQA
 from chainercv.datasets.directory_parsing_label_dataset import DirectoryParsingLabelDataset  # NOQA
+from chainercv.datasets.kitti.kitti_bbox_dataset import KITTIBboxDataset  # NOQA
+from chainercv.datasets.kitti.kitti_utils import kitti_bbox_label_colors  # NOQA
+from chainercv.datasets.kitti.kitti_utils import kitti_bbox_label_names  # NOQA
+from chainercv.datasets.kitti.kitti_utils import kitti_date_lists  # NOQA
```

Comment:
Do we need to expose this object at this namespace?
We only expose `_label_names` and `_colors` here.